pyproject.toml

```diff
@@ -148,24 +148,17 @@ python_version = 3.11
 disable_error_code = [
     "arg-type",
     "attr-defined",
-    "has-type",
     "import-untyped",
     "index",
     "list-item",
     "misc",
     "no-redef",
     "no-untyped-call",
     "no-untyped-def",
-    "override",
```

Comment:
This one needed a code change, base class used a different Request type in `lms/services/vitalsource/_client.py`.